packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import {
+  css,
+  autobind,
+  BaseComponent
+} from '../../Utilities';
+import { IOverflowSetProps } from './OverflowSet.Props';
+import { IconButton } from '../../Button';
+import { FocusZone, FocusZoneDirection } from '../../FocusZone';
+
+const styles: any = require('./OverflowSet.scss');
+
+export class OverflowSet extends BaseComponent<IOverflowSetProps, null> {
+
+  public render() {
+    let { items, overflowItems } = this.props;
+    return (
+      <FocusZone className={ css('ms-OverflowSet', styles.root) } direction={ FocusZoneDirection.horizontal } role='menubar' >
+        { items && this._onRenderItems(items) }
+        { overflowItems.length && this._onRenderOverflowButton(overflowItems) }
+      </FocusZone>
+    );
+  }
+
+  @autobind
+  private _onRenderOverflowButton(items) {
+    let { iconProps = {} } = this.props;
+    iconProps.iconName = iconProps.iconName ? iconProps.iconName : 'More';
+    return (
+      <IconButton
+        className={ css(styles.overflowIcon) }
+        iconProps={ iconProps }
+        menuIconProps={ { iconName: null } }
+        menuProps={ {
+          items: items
+        } }
+      />
+    );
+  }
+
+  @autobind
+  private _onRenderItems(items) {
+    return items.map((item, i) => {
+      let key = item.key ? item.key : i;
+      let onRender = item.onRender ? item.onRender : this.props.onRenderItem;
```

Comment:
This won't work;

onRender has a signature (item) => JSX.Element
onRenderItem has a signature (props, defaultRender) => JSX.element

Youre passing item and index, which satisfies neither.